src/core/evaluator.js

```diff
@@ -422,8 +422,16 @@ class PartialEvaluator {
         smask,
         isolated: false,
         knockout: false,
+        optionalContent: null,
       };
 
+      if (dict.has("OC")) {
```

Comment:
All form XObjects can have an `OC` entry, not just transparency groups that have a `Group` entry. Then you may need to pass the parsed `OC` value also to `paintFormXObjectBegin/End` and handle it in canvas.js.

Reply:
Good catch, I'll just insert a `beginMarkedContentProps` before the form object so I don't have to handle this for both.